test/unit/menu/CompasCompareIED.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import CompasCompareIEDPlugin from '../../../src/menu/CompasCompareIED.js';
+
+describe('Compas Compare IED Plugin', () => {
+  if (customElements.get('compas-compare-ied') === undefined)
+    customElements.define('compas-compare-ied', CompasCompareIEDPlugin);
+
+  let plugin: CompasCompareIEDPlugin;
+  let doc: XMLDocument;
+
+  beforeEach(async () => {
+    plugin = await fixture(html`<compas-compare-ied></compas-compare-ied>`);
+    doc = await fetch('/test/testfiles/menu/compare-ied-changed.scd')
+      .then(response => response.text())
+      .then(str => new DOMParser().parseFromString(str, 'application/xml'));
+  });
+
+  describe('show template project selection dialog', () => {
+    beforeEach(async () => {
+      plugin.doc = doc;
+      plugin.run();
+      await plugin.requestUpdate();
+    });
+
+    it('after closing the dialog everything set to undefined', async () => {
```

Comment:
The description does not fit 100%. It would be either ...everything stays undefined... or the starting condition must be different as you do not change anything.

Reply:
Changing the description is a good idea. It's more to make sure variable aren't changed when in this "mode".